data/strings/types_strings.txt

```diff
@@ -2011,6 +2011,10 @@
     zh-Hans = 图书馆
     zh-Hant = 圖書館
 
+  [type.amenity.loading_dock]
+    en = Loading dock
```

Comment:
Please run `DELIM=" = " tools/unix/generate_localizations.sh "Loading Dock"` to generate all translations.

Reply:
I want to say that I appreciated helpful fail message about submodules!

Reply:
Oh, and I did this.

@biodranik Should be ready now for review.

Reply:
1. Looks like you didn't commit your changes. 
2. Please amend and split commit so the first one contains txt changes, and the second has all regenerated files with the title `[strings] Regenerated`

Reply:
1. it was commited and squashed into a single commit (and here I in turn used rebase...)
2. done

Reply:
I still don't see all translations in the first commit.

Reply:
Is it about adding translations for every single language? In such case I am unable to do this, as I do not speak for example Arabic and I am unable to even verify translation provided by someone else or automatic translator.

Reply:
After inserting automated translations we can ask our translators team to review them.

Reply:
Weird. BTW, https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md was not really mentioning this. There is "In some cases automatically translated strings are better than no translation at all." but it has not seemed to be likely to be applying here... Especially as malformed translation can be easily missed unlike missing one.

Reply:
There is no big harm in this specific translation's issues. And our translation team has increased since those times when we wrote TRANSLATIONS.md )